openedx/core/djangoapps/content/course_overviews/management/commands/tests/test_generate_course_overview.py

```diff
@@ -0,0 +1,55 @@
+# pylint: disable=missing-docstring
+from openedx.core.djangoapps.content.course_overviews.management.commands import generate_course_overview
+from openedx.core.djangoapps.content.course_overviews.models import CourseOverview
+from xmodule.modulestore.tests.django_utils import ModuleStoreTestCase
+from xmodule.modulestore.tests.factories import CourseFactory
+
+
+class TestGenerateCourseOverview(ModuleStoreTestCase):
```

Comment:
Makes sense.  I've added `test_invalid_key`, `test_not_found_key`, and `test_no_params` to catch the edge cases.